web/src/recent_topics_ui.js

```diff
@@ -1251,25 +1255,23 @@ export function initialize() {
         popovers.show_user_info_popover(this, user);
     });
 
-    $("body").on("keydown", ".on_hover_topic_mute", ui_util.convert_enter_to_click);
-
-    $("body").on("click", "#recent_topics_table .on_hover_topic_unmute", (e) => {
-        e.stopPropagation();
-        const $elt = $(e.target);
-        const topic_row_index = $elt.closest("tr").index();
-        focus_clicked_element(topic_row_index, COLUMNS.mute);
-        muted_topics_ui.mute_or_unmute_topic($elt, false);
-    });
-
-    $("body").on("keydown", ".on_hover_topic_unmute", ui_util.convert_enter_to_click);
+    $("body").on(
+        "keydown",
+        ".on_hover_topic_unmute,.on_hover_topic_mute",
+        ui_util.convert_enter_to_click,
+    );
```

Comment:
Similarly, I preferred the previous approach of having parallel code where it would be explicit in the call into `muted_topics_ui` what state is requested.